README.md

```diff
@@ -17,7 +17,7 @@ It will be FLL-Ready (as soon as the basic feature set is fully implemented and
 Also it will help you to improve your movements through showing important data in the screen while driving. (Not implemented yet)
 
 ### How to use CharlieOSX with your robot:
-1. Install [Ev3dev](ev3dev.org/docs/getting-started) or [Pybricks](https://docs.pybricks.com/en/latest/start_ev3.html), with pybricks being the recommended one
```

Comment:
The download from LEGO Education has a slightly newer version of `pybricks-micropython` so it is probably better to recommend it at this point in time. The current ev3dev release has a beta version of `pybricks-micropython` but it can be upgraded to the final version via `apt`.

Reply:
Also, as far as I am concered you need to use Pybricks if you want to be FLL confirm, because (from the last time I read the rules) only official Lego software is allowed